distribution/test/itests/test-itests-alliance/src/test/java/alliance/test/itests/TestFederation.java

```diff
@@ -56,7 +58,27 @@
  */
 @RunWith(PaxExam.class)
 @ExamReactorStrategy(PerClass.class)
-public class TestNsili extends AbstractIntegrationTest {
+public class TestFederation extends AbstractIntegrationTest {
```

Comment:
In that case, I propose that we align all of Alliance to `*Test.java` so we can maintain consistency and eliminate any confusion.